.github/workflows/build.yml

```diff
@@ -3,15 +3,17 @@ on:
   push:
     branches:
       - 'main'
+  schedule:
+  - cron: "0 0 * * 0"
 
 jobs:
   build:
     runs-on: ubuntu-latest
     env:
       PKR_VAR_aws_access_key: ${{ secrets.AWS_ACCESS_KEY_ID }}
       PKR_VAR_aws_secret_key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-      #Targets AWS accounts for Jenkins AMI's--in this case tvm-ci-prod and terraform-tvm
-      PKR_VAR_target_accounts: '["477529581014", "649843420731"]'
+      #Targets AWS accounts for Jenkins AMI's--in this case tvm-ci-prod, terraform-tvm, and octoml-sandbox
+      PKR_VAR_target_accounts: '["477529581014", "649843420731", "186900524924"]'
```

Comment:
are we able to just publish these rather than posting to octo-specific account?

Reply:
I don't see why not--I'll push up a commit that does this.